### ISSUE TITLE
Add missed tests for SIMD with usize and isize type

### ISSUE THREAD

**Issue by JesseCodeBones** [AssemblyScript/assemblyscript#2466]
Hi Maintainer,

I found there is no tests for v128 with type `usize` and `isize`, I will prepare a PR for that.
please let me know if the tests are suitable or if I missed anything.

Regards

**Comment by MaxGraey** [comment]
I'm not sure using `isize` / `usize` parameter types for SIMD was a good idea. Some wasm ops may return different results in wasm32 / 64 mode. Like `v128.bitmask<usize>(...)` is not the same.

I more prefer remove supporting isize / usize for v128. @dcodeIO wdyt?

**Comment by JesseCodeBones** [comment]
@MaxGraey 
True, I add the condition `if (process.arch == "wasm64")` to figure out is it is wasm64

**Comment by JesseCodeBones** [comment]
if these type need to be removed from SIMD feature, should we add `/** @deprecated */` for these function signature firstly, as this kind of signatures already are apart of public APIs? WDYT?

**Comment by JesseCodeBones** [comment]
Related with PR [2491](https://github.com/AssemblyScript/assemblyscript/pull/2491).
